overlord/snapstate/snapstate_install_test.go

```diff
@@ -4020,3 +4020,42 @@ func (s *snapmgrTestSuite) TestHasAllContentAttributes(c *C) {
 	_, err = snapstate.HasAllContentAttrs(s.state, "some-snap", []string{"some"})
 	c.Assert(err.Error(), Equals, `expected 'content' attribute of slot 'bad-content-slot' (snap: 'some-snap') to be string but was int`)
 }
+
+func (s *snapmgrTestSuite) TestInstallPrereqIgnoreConfInSameChange(c *C) {
```

Comment:
Nitpick, I'd use the word "Conflict" in the test name, "Conf" may get confused with config-related stuff.